### ISSUE TITLE
On the /request page, separate "Received" and "Accepted" into two separate tabs

### ISSUE THREAD

**Issue by FlacoJones** [OpenQDev/OpenQ-Frontend#1716]
EthDenver is going to have one account with 1,800 tax forms

As they accept one document, i think we should, at the very least, remove it to a separate tab

Similar to the Tabs in bounty details page, lets have a "Pending" and an "Accepted" tab

**Comment by ArcAnya** [comment]
Is this issue then relevant as well here? #1647

**Comment by Christopher-Stevers** [comment]
> Is this issue then relevant as well here? #1647

Yeah, I think the implementation for both of these will be post eth denver though. For eth denver will be able to go directly to the page the request is on from the email they receive, as well as search.  Otherwise I think we'd have to make API changes, which I'm not keen on pushing atm.

**Comment by Christopher-Stevers** [comment]
Will return to this after eth denver.
